lib/debase.rb

```diff
@@ -1,5 +1,10 @@
 if defined?(RUBY_ENGINE) && 'rbx' == RUBY_ENGINE
   require 'debase/rbx'
+elsif defined?(RUBY_ENGINE) && RUBY_ENGINE == 'truffleruby'
```

Comment:
This backend is not supposed to run with truffleruby

Reply:
If the changes in this file are problematic, I can remove them or redo them.
The most critical part is that the debase C extensions are not attempted to be compiled on non-CRuby (because that would just fail).